### ISSUE TITLE
error messages in console

### ISSUE THREAD

**Issue by Lefthandmedia** [gadgetto/UserImport#11]
Sometimes i get these messages in the console
```
Please enter the subject for the email notification.
Please enter the body for the email notification.
Import failed. Please check error messages.
```
but not all of the time. Naturaly the subject and body are entered under the tab 'Notification template'.
The strange part is that it's not all the time. it looks like when i go to the tabs 'Notification Template' and then return to 'Users' the import works

**Comment by gadgetto** [comment]
Sorry for my (very) late response!
I can't duplicate this problem.

If the problem occurs, do you see any JavaScript errors in browser console log?

**Comment by gadgetto** [comment]
Fixed in MODX2 release 1.2.0-pl & MODX3 release 2.0.0-pl
